.github/workflows/test-git-setup.yml

```diff
@@ -0,0 +1,31 @@
+name: test-git-setup
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-git-setup.yml'
+      - 'git/setup/**'
+  push:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/test-git-setup.yml'
+      - 'git/setup/**'
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./git/setup
+      - name: validate git user
+        run:
+          test $(git config user.name) = "obltmachine"
```

Comment:
and the env variables too?

Reply:
https://github.com/elastic/oblt-actions/pull/6/commits/4050449805aa34d46fb3fb8a098598ed22412d7b